### PR TITLE
lChange element types to be more semantic, add aria labels, roles and …

### PR DESCRIPTION
**Pull request by zenmcmillan**
…type attributes, add or improve existing alt attribute description

## Necessary checkmarks:

    [] All Tests are Passing in all environments

    [x] The code will run locally

## Type of change

    [] New feature
    [] Bug Fix
    [x] Refactor
    [] Testing

## Description of Change:

    - Changed elements to be more semantic
    - Added aria labels and roles on elements when deemed necessary
    - Added or improved alt attribute description for images
    
## Requested feedback: 
    
    I'd like feedback on... 

## Check the correct boxes

    [x] This broke nothing
    [] This broke some stuff
    [] This broke everything

## Testing Changes

    [x] No Tests have been changed
    [] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

    [x] My code has no unused/commented out code
    [x] My code has no binding.pry's
    [x] I have reviewed my code
    [] I have commented my code, particularly in hard-to-understand areas
    [x] I have fully tested my code

## Next Steps?

### CODE REVIEW

**Comment by skim1027** [comment]
Hi Zen - just wanted to double check that with this change, have you re-run the cypress test, and does it pass as is?

**Comment by zenmcmillan** [comment]
I'm making some changes now to ensure that all the tests are passing

**Comment by zenmcmillan** [comment]
So there was one test that wasn't passing. I made some changes and its now
passing. It's been pushed up to the existing pull request

File: src/Components/SavedRecipeCard/SavedRecipeCard.tsx [hunk 1]
```diff
@@ -13,20 +13,23 @@ const SavedRecipeCard: React.FC<SavedRecipeCardProps> =
...
+      <button className='show_details_btn' 
+      onClick={toggleDropdown}
+      aria-expanded={showDropdown}>
       {showDropdown ? 'Close Details' : 'Show Details'}
       </button>
       {showDropdown === true && (
-        <div className='dropdown'>
-          <h4>Ingredients</h4>
+        <div className='dropdown'
+          aria-hidden={!showDropdown}>
```
Comment (Jnguyen615) [inline src/Components/SavedRecipeCard/SavedRecipeCard.tsx:25]: This is interesting here, I haven't worked with react and accessibility so this is cool to see how it's done.

**Review by Jnguyen615 — APPROVED** [review]
I like seeing some of the differences for accessibility in react in comparison to javascript. Nice job!

**Pull request merged**